Provide a way to generate docs for contributors

### OpenTofu Version

```shell
before
```


### OpenTofu Configuration Files

```hcl
...tofu config...
```


### Debug Output

-

### Expected Behavior

<s>The make website targets here are expected to generate the documentation website in the local environment.</s>  
https://github.com/opentofu/opentofu/blob/60a9db0a43b574c505b6a43614852458ff0c59c5/Makefile#L40-L53

Those make targets are leftovers from the Terraform documentation. Currently, the website is run by https://github.com/opentofu/opentofu.org repository. How to run the website documentation locally is described [here](https://github.com/opentofu/opentofu.org/blob/main/README.md). 

<s>In a general sense, We should be able to provide a way for developers to be able to run the MDX website locally so that they can contribute to the docs and check their changes before creating a PR.</s> It would be helpful to document how to use opentofu.org GitHub repository to run the documentation locally. 

The [Terraform website/ Makefile](https://github.com/hashicorp/terraform/blob/v1.5.7/website/Makefile) has been removed because it was using a HashiCorp specific docker images, see https://github.com/hashicorp/terraform/blob/ee58ac1851c8a433005df9863ed47796a9f6b5e7/website/Makefile#L19-L20

### Actual Behavior

There is no Makefile in the [website/ directory.](https://github.com/opentofu/opentofu/tree/main/website), that breaks the `make website` targets. 

There are no clear instructions on how to generate the website locally. 


### Steps to Reproduce

```
% make website
/Library/Developer/CommandLineTools/usr/bin/make -C website website
make[1]: *** No rule to make target `website'.  Stop.
make: *** [website] Error 2
```
```
% make website/build-local 
/Library/Developer/CommandLineTools/usr/bin/make -C website website/build-local
make[1]: *** No rule to make target `website/build-local'.  Stop.
make: *** [website/build-local] Error 2
```
```
% make website/local
/Library/Developer/CommandLineTools/usr/bin/make -C website website/local
make[1]: *** No rule to make target `website/local'.  Stop.
make: *** [website/local] Error 2
```

### Additional Context

_No response_

### References

_No response_

--- a/Makefile
+++ b/Makefile
@@ -37,21 +37,6 @@ staticcheck:
 exhaustive:
 	"$(CURDIR)/scripts/exhaustive.sh"
 
-# Run this if working on the website locally to run in watch mode.
-.PHONY: website
-website:
-	$(MAKE) -C website website
-
-# Use this if you have run `website/build-local` to use the locally built image.
-.PHONY: website/local
-website/local:
-	$(MAKE) -C website website/local
-
-# Run this to generate a new local Docker image.
-.PHONY: website/build-local
-website/build-local:
-	$(MAKE) -C website website/build-local
-
 # Run license check
 .PHONY: license-check
 license-check:

--- a/website/README.md
+++ b/website/README.md
@@ -8,16 +8,6 @@ You can [submit an issue](https://github.com/opentofu/opentofu/issues/new/choose
 
 Click **Edit this page** at the bottom of any OpenTofu website page to go directly to the associated markdown file in GitHub.
 
-## Validating Content
-
-Content changes are automatically validated against a set of rules as part of the pull request process. If you want to run these checks locally to validate your content before committing your changes, you can run the following command:
-
-```
-npm run content-check
-```
-
-If the validation fails, actionable error messages will be displayed to help you address detected issues.
-
 ## Modifying Sidebar Navigation
 
 You must update the sidebar navigation when you add or delete documentation .mdx files. If you do not update the navigation, the website deploy preview fails.
@@ -30,7 +20,13 @@ To update the sidebar navigation, you must edit the appropriate `nav-data.json` 
 
 ## Previewing Changes
 
-Coming soon: Documenting the development process for the documentation website repo.
+Currently, you can preview your changes through the [opentofu/opentofu.org](https://github.com/opentofu/opentofu.org/blob/main/README.md) repository.
+
+Follow the [Getting Started](https://github.com/opentofu/opentofu.org/blob/main/README.md#getting-started) guide.
+If you would like to fetch the documentation from another repository and another branch, you can run the make command with REPO_URL and BRANCH arguments before starting the development server:
+```
+make REPO_URL="<your_forked_repo_url>" BRANCH="<branch_name>"
+```
 
 ## Deploying Changes
 

